data_structures/stacks/next_greater_element.py

```diff
@@ -17,12 +17,12 @@ def next_greatest_element_slow(arr: list[float]) -> list[float]:
     arr_size = len(arr)
 
     for i in range(arr_size):
-        next: float = -1
+        _next: float = -1
```

Comment:
```suggestion
        next_: float = -1
```
https://www.datacamp.com/tutorial/role-underscore-python